### ISSUE TITLE
[FEATURE] Comment Indentation Configurability 

### ISSUE THREAD

**Issue by dthiot** [mvextensions/mvbasic#111]
Add a workspace parameter "Comment Indentation" that would have options for none, left, margin,  or code.  None means that comments are not indented and left where they are.  Left means that comments are moved all the way to the left (zero margin).  Margin means that code is indented based on the margin setting.  Code means that the comments are indented to the same level as the next line of code is indented.  That means after a line of code that causes a new indentation, the comment is indented to the new level.

**Comment by kpowick** [comment]
I'm actually making good progress on this suggestion.  I hope to have it implemented today. 😀

**Comment by kpowick** [comment]
I've implemented this feature as new user setting `MVBasic.indentComment`.  I thought the name followed well with the already existing `MVBasic.indent`.

The setting uses the values suggested by @dthiot , but I changed "none" to "ignore".

Possible values for the setting are: 

`left` - indent comment line to margin position 0.

`margin` - Indent comment line to the position defined by `MVBasic.margin` .

`code` - Indent comment line to the same position as code indentation.

`ignore` - Do not change the indentation of the comment line. 

The default setting is `ignore` because this is how 2.0.8 currently treats comment lines.  I didn't want any surprises with the next release of the extension.

<img width="524" alt="Screen Shot 2020-06-12 at 15 54 07" src="https://user-images.githubusercontent.com/1646753/84543067-a3d05e80-acc8-11ea-9d08-8a1e3eba7eab.png">
